uobtheatre/utils/test/test_utils_utils.py

```diff
@@ -4,3 +4,10 @@
 def test_create_short_uuid_len():
     shortuuid = create_short_uuid()
     assert len(str(shortuuid)) == 12
+
+
+def test_create_short_uuid_uniqueness():
+    shortuuid_one = create_short_uuid()
+    shortuuid_two = create_short_uuid()
+
+    assert shortuuid_one != shortuuid_two
```

Comment:
I feel like this also arrived on a different branch :thinking:

Reply:
This was part of the problem solving, will remove!

Reply:
Oh sorry I forgot this was in this branch! No harm in keeping it tbh

Reply:
is this test necessary?
I can see why it's there and it's harmless to have it - but surely we are assuming the module's testing is rigorous and even if it isn't this single assertion wouldn't have enough rigour to be valid anyway.

If it's just for coverage tho then i'm totally for it xox

Reply:
Its not needed but its a nice to have. Until this pr, if someone had just changed `create_short_uuid` to return `1` all tests would have passed. This at least checks a very low level of uniqueness.

Reply:
fair xx